src/DynamoCoreWpf/ViewModels/Core/DynamoViewModel.cs

```diff
@@ -95,6 +95,22 @@ public PreferenceSettings PreferenceSettings
             get { return Model.PreferenceSettings; }
         }
 
+        /// <summary>
+        /// This property will be used by the Preferences screen to store and retrieve all the seetings from the expanders
+        /// </summary>
+        public List<ExpanderSettings> ExpandersSettings
```

Comment:
Why dont we store this in Preferences View Model since it does not directly affect DynamoViewModel?